templates/documents/general_view.html

```diff
@@ -109,16 +109,24 @@
 <table valign="top" width="100%">
     <tr>
         <td>
+            <div style="margin-bottom: 6px;padding-bottom: 6px;border-bottom:3px solid gray;">
+            <a class="css_button">{$file->get_url_web()}</a>
+            <label for='chkShow' class="text" style="margin-left:20px;"><b>{xl t='View contents'}</b>
```

Comment:
It confused me. I initially thought it was a bug (I assumed View contents meant to see the actual document :) ). And then I wondered why I could see the document whether I clicked View contents or not. And then I realized View contents meant to view the "Details". And then all was good.